Smarty/templates/modules/Documents/pdfViewer.html

```diff
@@ -245,25 +270,31 @@
                     <i class="fa-sharp fa-solid fa-angle-right"></i>
```

Comment:
I see we have more of these <i>, how were they working before we added the bootstrap library?  or they are not related. Do these come from fontawesome ??

Reply:
@joebordes - They are font awesome, but we have them in this file `Smarty/templates/modules/Documents/all.min.css`